src/scribe_data/wikipedia/process_wiki.py

```diff
@@ -339,10 +337,9 @@ def gen_autosuggestions(
     if isinstance(ignore_words, str):
         words_to_ignore = [ignore_words]
     elif ignore_words is None:
-        words_to_ignore += []
+        words_to_ignore = []  # Fixed from +=
```

Comment:
Can we lower case `Fixed` as this is an inline comment. Inline should be lower case and no period, and own line should be capitalized with a period :)